src/components/form.js

```diff
@@ -1,14 +1,23 @@
+import React from 'react';
 import './form.css';
+import { useDispatch } from 'react-redux';
+import { addBook, Book } from '../redux/books/books';
 
-const Form = () => (
-  <form className="add-form">
-    <h2 className="add-Title">ADD NEW BOOK</h2>
-    <div className="input-area">
-      <input type="text" placeholder="Book title" />
-      <input type="text" placeholder="Category" />
-      <button type="submit"><span className="ADD-BOOK">ADD BOOK</span></button>
-    </div>
-  </form>
-);
+const Form = () => {
+  const dispatch = useDispatch();
+  const submitBookToStore = () => {
+    dispatch(addBook(Book));
```

Comment:
- [ ] The Book object should be created using the local state and should look similar to this.
```
const Book = {
        id, // make sure it's unique
        title,
        author
    }
```